src/fidesops/api/v1/endpoints/user_endpoints.py

```diff
@@ -49,6 +59,34 @@ def create_user(
     return user
 
 
+@router.get(
+    urls.USERS,
+    dependencies=[Security(verify_oauth_client, scopes=[USER_READ])],
+    response_model=Page[UserResponse],
+)
+def get_users(
+    *, db: Session = Depends(deps.get_db), params: Params = Depends()
+) -> AbstractPage[FidesopsUser]:
+    """Returns a paginated list of all users"""
+    logger.info(f"Returned a paginated list of all users.")
+    return paginate(FidesopsUser.query(db), params=params)
```

Comment:
Paginated queries need a default order passed into the query too, so that each page can be selected consistently. See: https://github.com/ethyca/fidesops/blob/main/src/fidesops/api/v1/endpoints/privacy_request_endpoints.py#L354

Reply:
I've added a bug for the remaining parts of the system where this is an issue here: https://github.com/ethyca/fidesops/issues/411